sonar-php-plugin/src/main/java/org/sonar/plugins/php/PhpIniSensor.java

```diff
@@ -53,7 +54,10 @@ public PhpIniSensor(CheckFactory checkFactory) {
 
   @Override
   public void describe(SensorDescriptor descriptor) {
-    descriptor.name("Analyzer for \"php.ini\" files");
+    descriptor
+      .name("Analyzer for \"php.ini\" files")
+      // don't target ".php" files, but it makes no sense to invoke this sensor for project without ".php" files
+      .onlyOnLanguage(Php.KEY);
```

Comment:
It would be worth having at least 1 integration test checking that the sensor is run.

Reply:
I added 2 integration tests, one assert the sensor is run, one assert the sensor is not run.